layouts/shortcodes/deprecationwarning.html

```diff
@@ -3,8 +3,9 @@
   <main>
     <div class="content deprecation-warning">
       <h3>
-        Documentation for Kubernetes {{ .Page.Param "version" }} is no longer actively maintained. The version you are currently viewing is a static snapshot.
-        For up-to-date documentation, see the <a href="{{ .Site.Params.currentUrl }}">latest</a> version.
+	 Kubernetes {{ .Param "version" }}
+	 {{ T "deprecation_warning" }}
+	 <a href="{{ .Site.Params.currentUrl }}">{{ T "latest_version" }}</a>
```

Comment:
you changed the name of 'latest_version' in the i18n file!

Reply:
@tengqm Thank you for the comment. I just noticed the mistake. I restored [latest_version] from [shortcode_latest_version].

Reply:
Thanks.